tasks/main.yml

```diff
@@ -4,12 +4,40 @@
   register: pecl_msgpack_result
   changed_when: "'installed' not in pecl_msgpack_result.stdout"
   failed_when: "'ERROR' in pecl_msgpack_result.stdout"
+  when: pecl_zmq_version == '0.5.7'
 
 - name: ensure php zmq extension is installed
   shell: yes '' | pecl install zmq-{{ pecl_zmq_version }}
   register: pecl_zmq_result
   changed_when: "'installed' not in pecl_zmq_result.stdout"
   failed_when: "'ERROR' in pecl_zmq_result.stdout"
+  when: pecl_zmq_version == '1.1.2'
```

Comment:
Only this version can be installed using pecl? Or it's because of PHP7?

Reply:
When use PHP7.1 , can't use pecl to install zmq-1.1.2, zmq-1.1.3, 
when use php5,  we can use pecl to install zmq-1.1.2, zmq-1.1.3.
should be php7 cause this problem.

Reply:
Okay, then let's use PHP version for this condition